extensions/amp-story/1.0/amp-story-interactive-binary-poll.js

```diff
@@ -203,7 +211,31 @@ export class AmpStoryInteractiveBinaryPoll extends AmpStoryInteractive {
       currOption.querySelector(
         '.i-amphtml-story-interactive-option-percentage-text'
       ).textContent = `${percentage}%`;
-      currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      setStyle(
+        currOption.querySelector(
+          '.i-amphtml-story-interactive-option-percent-bar'
+        ),
+        'transform',
+        `scaleX(${percentage * 0.01 * 2})`
+      );
+
+      const textContainer = currOption.querySelector(
+        '.i-amphtml-story-interactive-option-text-container'
+      );
+
+      textContainer.setAttribute(
+        'style',
+        `transform: translateX(${
+          percentage < MIN_HORIZONTAL_PADDING
+            ? MIN_HORIZONTAL_PADDING * (index === 0 ? -1 : 1)
+            : (percentage - 50) * (index === 0 ? 1 : -1)
```

Comment:
Not convinced on this, mainly because the edges around MIN_HORIZONTAL_PADDING are not continuous: check
https://www.desmos.com/calculator/knvwjvr1lk where I wrote the function that maps percentage to translateX value.

I'm a bit worried about how different 20% and 21% will look (or whatever threshold we use). Can you add that to the demo?

Reply:
(Also, you might have to invert the translate for RTL)

Reply:
@mszylkowski I see what you mean. Thank you for providing the graph. 
I'll update these values so there is not a big shift when the numbers move beyond the threshold.

@gmajoulet will do.

Reply:
@mszylkowski This has been revised per our discussion. Thank you for your help thinking through this.
@gmajoulet done.